client/lib/site/test/test.js

```diff
@@ -1,101 +1,67 @@
-var assert = require( 'assert' );
-
-var Site = require( '../' );
-
-describe( 'Calypso Site', function() {
-	describe( 'setting attributes', function() {
-
-		it( 'attribute changed', function() {
-			var site = Site( {
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+import sinon from 'sinon';
+
+import Site from '..';
+
+describe( 'Calypso Site', () => {
+	describe( 'setting attributes', () => {
+		let site;
+		beforeEach( () => {
+			site = Site( {
 				ID: 1234,
 				name: 'Hello',
 				description: 'Hunting bugs is fun.'
 			} );
+		} );
 
+		it( 'attribute changed', () => {
 			site.set( { name: 'Goodbye' } );
-
-			assert( site.name === 'Goodbye' );
+			expect( site.name ).to.equal( 'Goodbye' );
```

Comment:
We don't have any preferred assertions style. You can leave this updated, but if you plan to modify more tests, please omit this kind of refactorings to save time.